hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -285,7 +286,11 @@ public int getConfigServerPort() {
   }
 
   public int getReconciliationTimeout() {
-    return getConf().getInt("mesos.reconciliation.timeout.seconds", DEFAULT_RECONCILIATION_TIMEOUT);
+    return getConf().getInt("mesos.reconciliation.timeout.seconds", DEFAULT_RECONCILIATION_TIMEOUT_SEC);
```

Comment:
Let's make them all consistent please. Some are sec, some are seconds. I understand this was not you, but this should be a simple cleanup.